queries/go/refactoring.scm

```diff
@@ -0,0 +1,16 @@
+;; Grabs all the local variable declarations.  This is useful for scope
+;; variable passing.  Which variables do we need to pass to the extracted
+;; function?
+((short_var_declaration
```

Comment:
Let me try these out to make sure its correct.

Reply:
Added tests to make validation of this easier. Thanks again for the test harness! Made fixing these so much easier.

Reply:
awsome!